hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/order/SigRequirements.java

```diff
@@ -723,6 +746,10 @@ private <T> SigningOrderResult<T> cryptoTransfer(
                             ? factory.forMissingToken()
                             : accountFailure(failure, factory);
                 }
+                mapWarmer.warmTokenRel(
+                        EntityNumVirtualKey.fromPair(
+                                EntityNumPair.fromLongs(
+                                        token.getTokenNum(), adjust.getSerialNumber())));
```

Comment:
@tinker-michaelj suggested that we would get the most bang for our buck by putting the `warm(...)` methods inside of handling for crypto transfers, so all warming added so far will only take effect on a crypto transfer. We can add warming to other code paths, of course, but this was the suggestion. It may be worth putting a debug log statement into `MapWarmer` so we know for sure each time we've tried to call `warm(...)` on an object.